src/electron/office/office_web_plugin.cc

```diff
@@ -275,6 +281,21 @@ blink::WebInputEventResult OfficeWebPlugin::HandleKeyEvent(
       return blink::WebInputEventResult::kNotHandled;
   }
 
+  const char16_t copy = 'c';
+  const char16_t paste = 'v';
+
+  if ((event.GetModifiers() == event.kControlKey ||
+       event.GetModifiers() == event.kMetaKey) &&
+      (*event.unmodified_text == copy || *event.unmodified_text == paste)) {
+    if (*event.unmodified_text == copy) {
+      return HandleCopyEvent();
+    } else if (*event.unmodified_text == paste) {
+      return HandlePasteEvent();
+    }
+    // This should never be hit
+    return blink::WebInputEventResult::kNotHandled;
+  }
+
```

Comment:
This code is redundant to the code immediately below it.

Just add a case for US_C and US_V below.